Cannot install on ubuntu 12.04.2 LTS without libusb-1.0-0-dev package.

When trying to install on `ubuntu 12.04.2 LTS` I got the following error:

``` bash
make: Entering directory `/var/www/node-powermate/node_modules/node-hid/build'
  CC(target) Release/obj.target/hidapi/hidapi/libusb/hid.o
../hidapi/libusb/hid.c:47:20: fatal error: libusb.h: No such file or directory
```

Installing the following package fixed the issue:

``` bash
sudo apt-get install libusb-1.0-0-dev
```

Installing the `dev` package did not work:

``` bash
sudo apt-get install libusb-dev
```

---

``` bash
peter@edgy:/var/www/node-powermate$ cat /etc/issue
Ubuntu 12.04.2 LTS \n \l

peter@edgy:/var/www/node-powermate$ uname -a
Linux edgy 3.2.0-32-generic #51-Ubuntu SMP Wed Sep 26 21:33:09 UTC 2012 x86_64 x86_64 x86_64 GNU/Linux
```

--- a/README.md
+++ b/README.md
@@ -7,6 +7,7 @@
 * Mac OS (I use 10.6.8) or Linux (kernel 2.6+) or Windows XP+
 * node.js v0.8
 * libudev-dev (Linux only)
+* libusb-1.0-0-dev (Ubuntu versions missing `libusb.h` only)
 * git
 
 ### Compile from source on Linux or OSX

